moz_security/tests/integration/analysis_auth_ip_geo/run/input/generate_auth.cfg

```diff
@@ -0,0 +1,21 @@
+filename = "generate_auth.lua"
+
+maxminddb_heka = {
+    databases = {
+        ["GeoIP2-City-Test.mmdb"] = {
```

Comment:
We are starting to get multiple copies of this DB in the repo, should we just start referencing the one here https://github.com/mozilla-services/lua_sandbox_extensions/blob/master/maxminddb/tests/GeoIP2-City-Test.mmdb